qiskit/test/mock/fake_backend.py

```diff
@@ -96,16 +96,13 @@ def properties(self):
 
         return BackendProperties.from_dict(properties)
 
-    def run(self, qobj):
+    def run(self, _):
         job_id = str(uuid.uuid4())
-        job = FakeJob(self, job_id, self.run_job, qobj)
+        job = FakeJob(self, job_id, self.run_job)
```

Comment:
Continuing from above this also tracks with storing the qobj in the job object constructed which matches how most providers and backend. We should store the qobj in the job so it can be accessed later in a true fake object. While not a necessity per the abstract class, most do this to make it easy to access the experiments input to the run.

Reply:
added back